### ISSUE TITLE
PORT in use for flask.

### ISSUE THREAD

**Issue by zeronone** [SNUArduinoLabs/coe_client#19]
I guess the following simple hack would work. In file `/etc/supervisor/conf.d/supervisor_webserver.conf`

```
command=fuser -n tcp -k 5000; python /home/pi/coe_client/src/webserver.py
```

**Comment by zeronone** [comment]
No this doesn't seems to work.

**Comment by cacyss0807** [comment]
Fixed by @zeronone : 946a1e45b7ddf501f2df50a3609244fcabd6856a
